### PR TITLE
DISTMONGO-200 Convert docs to Markdown (6.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
Configure MkDocs for doc build
Configure Render for docs hosting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
 
 Percona Distribution for MongoDB consists of the following components:
 
-- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.4/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
+- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.2/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
```
Comment (nastena1606) [inline README.md]: Version is 6.0

File: README.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
...
 - [Percona Backup for MongoDB](https://www.percona.com/doc/percona-backup-mongodb/index.html) - a distributed, low-impact solution for achieving consistent backups of MongoDB sharded clusters and replica sets.
 
-This repository contains the source files for the [Percona Distribution for MongoDB documentation](https://www.percona.com/doc/percona-distribution-for-mongodb/4.4/index.html).
+This repository contains the source files for the [Percona Distribution for MongoDB documentation](https://www.percona.com/doc/percona-distribution-for-mongodb/4.2/index.html).
```
Comment (nastena1606) [inline README.md]: Same here, version is 6.0

File: README.md [hunk 3]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
...
 
 The documentation is written in [reStructured text markup language](https://docutils.sourceforge.io/rst.html) and is created using [Sphinx Python Documentation Generator](https://www.sphinx-doc.org/en/master/).
 
 ## Contributing
 
-We welcome all contributions and are always looking for new members that are as dedicated to serving the community as we are. You can reach out to us using our [forums](https://forums.percona.com/c/mongodb/percona-distribution-for-mongodb/43) and [Discord](https://discord.gg/mQEyGPkNbR).
+We welcome all contributions and are always looking for new members that are as dedicated to serving the community as we are. You can reach out to us using our [forums](https://forums.percona.com/c/postgresql/25) and [Discord](https://discord.gg/mQEyGPkNbR).
```
Comment (nastena1606) [inline README.md]: The URL should lead to pdmb forum

File: README.md [hunk 4]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
...
 
-For how to contribute to documentation, read the [Contributing guide](https://github.com/percona/distmongo-docs/blob/4.4/CONTRIBUTING.md).
+For how to contribute to documentation, read the [Contributing guide](https://github.com/percona/distmongo-docs/blob/4.2/CONTRIBUTING.md).
```
Comment (nastena1606) [inline README.md]: 6.0 contributing guide

File: _resource/overrides/main.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+{# Import the theme's layout. #}
+{% extends "base.html" %}
+
+{%- macro relbar2 () %}
+<div class="md-relbar2__inner md-grid">
+<div class="md-content">
+  <article class="md-content__inner md-typeset" role="main">    
+    <h4>Contact Us </h4>
+       <p>For free technical help, visit the Percona <a class="reference external" href="https://forums.percona.com/c/postgresql/25?utm_campaign=Doc%20pages" target="_blank">Community Forum</a>.<br>
```
Comment (nastena1606) [inline _resource/overrides/main.html]: Please change the link to pdmdb one instead of postgresql

File: mkdocs-base.yml [hunk 6]
```diff
@@ -0,0 +1,126 @@
+# MkDocs configuration for builds with material theme
+
+site_name: Percona Distribution for MongoDB 6.0
+site_description: Documentation
+site_author: Percona LLC
+copyright: Percona LLC, &#169; 2022
+
+repo_name: percona/psmdb-docs
+repo_url: https://github.com/percona/distmongo-docs
+edit_uri: edit/4.2/docs/
```
Comment (nastena1606) [inline mkdocs-base.yml]: The url should be 6.0 here

File: mkdocs.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# MkDocs configuration for builds with material theme
+
+INHERIT: mkdocs-base.yml
+
+site_url: "https://docs.percona.com/percona-distribution-for-mongodb/"
```
Comment (nastena1606) [inline mkdocs.yml]: I believe we need a version here too

Comment (alina-derkach-oaza) [reply mkdocs.yml]: Updated

File: mkdocs-base.yml (hunk 6 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: Updated

File: _resource/overrides/main.html (hunk 5 above)
Comment (alina-derkach-oaza) [reply _resource/overrides/main.html]: Updated

File: README.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

**Pull request merged**